README.md

```diff
@@ -201,6 +202,14 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 ###xSQLServerPowerPlan
 * **Ensure**: (key) An enumerated value that describes if Min and Max memory is configured
 
+### xSqlAlias
+
+* **SQLServerName**: The name of Alias (e.g. svr01\inst01).
+* **ServerName**: The name of real server.
+* **Protocol**: The protocol of either tcp or np (named pipes).
+* **RetryCount**: Maximum number of retries to check HA group existency.
+* **TCPPort**: The tcp port of the instance.
+
 ## Versions
 
 <<<<<<< HEAD
```

Comment:
You should also add a section like this

## Unreleased
* Resources added
    * xSqlAliase